ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/Cli.scala

```diff
@@ -229,11 +229,6 @@ object Cli {
         success
     )
 
-    opt[Unit]("static-time")
-      .optional()
-      .action((_, c) => c.copy(staticTime = true))
-      .text("Must be enabled if the ledger is using static time")
```

Comment:
Are we sure it's not relied upon in integrations? If it is, we should rather deprecate it.

Reply:
I would prefer we break and have to fix those integrations. The flag already does nothing.

Reply:
Afaik we have no static tests in vmbc, nor in pkv. This was recently introduced so I would not expect it to be used anywhere.